Formula/traildb.rb

```diff
@@ -0,0 +1,21 @@
+
+class Traildb < Formula
+  desc "Blazingly-fast database for log-structured data"
+  homepage "http://traildb.io"
+  url "https://github.com/traildb/traildb/archive/0.5.tar.gz"
+  sha256 "4d1b61cc7068ec3313fe6322fc366a996c9d357dd3edf667dd33f0ab2c103271"
+
+  depends_on "libarchive"
+  depends_on "homebrew/boneyard/judy"
```

Comment:
This was just moved to boneyard, see https://github.com/Homebrew/homebrew-core/issues/1562. The reason for the move is that it's unmaintained. But it's a stable smallish C library that implements a single data structure. It's unmaintained because it's "done". What should we do then? Move it back from boneyard?